src/main/java/com/jcabi/w3c/DefaultCssValidator.java

```diff
@@ -102,6 +113,31 @@ private ValidationResponse processed(final String css) throws IOException {
     }
 
     /**
+     * Check if response from W3C contains some bad status.
+     * @param response Response from W3c.
+     * @throws IOException when has some bad status.
+     */
+    private void assertThatDoesNotHasBadHttpStatusAt(final Response response)
+        throws IOException {
+        final List<Integer> badStatuses = Arrays.asList(
```

Comment:
@dmzaytsev I disagree, but I'll refactor it. A name like this prevent me to write a javadoc, especially for a private method that is seen only by who is reading the code.